README.md

```diff
@@ -14,6 +14,7 @@ ___
 * RefApp 2.x Workflow Tests (Chrome) [BDD Chrome](https://ci.openmrs.org/browse/CONTRIB-QA) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/CONTRIB-QA)
   * [![RefApp 2.x Clinical Visit Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml)
   * [![RefApp 2.x Vitals And Triaging Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-vitals-and-triaging.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-vitals-and-triaging.yml) 
+  * [![RefApp 2.x Encounter Management Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-encounter.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-encounter.yml)
   * Tests in development:
     * Managing User Accounts
```

Comment:
Do we still need this here on line 19

Reply:
> What does this test basically do

Just as the commit title message states, the test against encounters that the clinician can do with the patient eg `Admission`

Reply:
What do you mean by an encounter in openmrs , How do you really understand what an encounter is in openmrs